bin/methods/setup.js

```diff
@@ -3,22 +3,44 @@ const path = require('path')
 const fs = require('fs')
 const prompt = require('prompt')
 const log = require('log-cb')
-const pixie = require('pixie')
+const configDefaults = require('../files/configDefaults.json')
 
 const CWD = process.cwd()
-const TEMPLATE_PATH = path.join(__dirname, '../files/cordlrrc.tmpl')
-const TEMPLATE = JSON.parse(fs.readFileSync(TEMPLATE_PATH))
+const packagePath = path.join(CWD, 'package.json')
 
 function setup () {
   prompt.get(['token', 'prefix'], function (err, result) {
     if (err) return log()(err)
 
     // Create and write config
-    const config = pixie.compile(TEMPLATE, result)
+    const config = Object.assign({}, configDefaults, { token: result.token, prefix: result.prefix })
     writeConfig(config)
   }).start()
 }
 
-function writeConfig (content) {
-  fs.writeFile(path.join(CWD, '.cordlrrc'), content, log('Configuration created'))
+function writeConfig (config) {
+  return new Promise((resolve, reject) => {
+    fs.readFile(packagePath, (er, data) => {
+      if (er) { // means the file doens't exist
+        const newPackage = { cordlr: config }
+        fs.writeFile(packagePath, JSON.stringify(newPackage, null), (err) => {
+          if (err) {
+            reject(err)
+          } else {
+            resolve(newPackage)
+          }
+        })
+      } else {
+        const existingPackage = JSON.parse(data)
+        const newPackage = Object.assign({}, existingPackage, { cordlr: config })
+        fs.writeFile(packagePath, JSON.stringify(newPackage), (err) => {
+          if (err) {
+            reject(err)
+          } else {
+            resolve(newPackage)
+          }
+        })
+      }
+    })
```

Comment:
I think this code could repeat less by just handling the data inside the `if` and not doing separate calls:

```js
let newPackage = JSON.parse(data)
if (!err) {
  const existingPackage = JSON.parse(data)
  Object.assing(newPackage, existingPackage)
}

fs.write(packagePath, JSON.stringify(newPackage), ...)
```